src/base/memory_monitor_sr.cpp

```diff
@@ -393,49 +523,42 @@ namespace cubperf
 /* APIs */
 int mmon_initialize (const char *server_name)
 {
+  int error = NO_ERROR;
   cubperf::mmon_Gl = new cubperf::memory_monitor (server_name);
 
   if (cubperf::mmon_Gl == nullptr)
     {
-      /* error case */
-      return -1;
+      error = ER_OUT_OF_VIRTUAL_MEMORY;
```

Comment:
`er_set` should be called where the error is set.
example is as below :  
```suggestion
      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1, sizeof (cubperf::memory_monitor));
      error = ER_OUT_OF_VIRTUAL_MEMORY;
```

Reply:
This module should be define error messages to describe each error case correctly.

Reply:
Done.